bin/build-hdfs

```diff
@@ -124,8 +123,8 @@ cp $PROJ_DIR/bin/hdfs-mesos $BUILD_DIR/$DIST/bin
 mkdir -p $BUILD_DIR/$DIST/lib
 cp $PROJ_DIR/hdfs-scheduler/build/libs/*-uber.jar $BUILD_DIR/$DIST/lib
 cp $BUILD_CACHE_DIR/$EXECUTOR.tgz $BUILD_DIR/$DIST
-mkdir -p $BUILD_DIR/$DIST/etc/hadoop
-cp $PROJ_DIR/conf/*.xml $BUILD_DIR/$DIST/etc/hadoop
+mkdir -p $BUILD_DIR/$DIST/conf
```

Comment:
Instead of using a yaml config here, what I would prefer to do is something similar to the following PR (which didn't get finished), https://github.com/mesosphere/hdfs/pull/123/. I would like to use environment variables or command line arguments (similar to Cassandra). These values could then be used to generate the *-site.xml configs used by HDFS.